docs/02-app/01-building-your-application/05-optimizing/01-images.mdx

```diff
@@ -18,7 +18,9 @@ related:
 
 </details>
 
-The Next.js Image component extends the HTML `<img>` element with:
+According to [Web Almanac](https://almanac.httparchive.org), images account for a huge portion of the typical website’s [page weight](https://almanac.httparchive.org/en/2022/media#images) and can have an sizable impact on your website's [LCP performance](https://almanac.httparchive.org/en/2022/performance#lcp-image-optimization).
+
+The Next.js Image component extends the HTML `<img>` element with features that make image optimization easier to achieve:
```

Comment:
"make image optimization easier to achieve" sounds like the component doesn't work in all cases or that it is not zero config.

Can we word this differently or drop it?

Reply:
> sounds like the component doesn't work in all cases or that it is not zero config.

Not at all how I intended it. Took a stab at another phrasing.